### PR TITLE
example tiled code

### PR DESCRIPTION
**Pull request by dylanmcreynolds**
Provides a sample notebook that shows how to communicate with the tiled server hosted for this project.

Also, splits requirements.txt files so that files used in development are in requirements-dev.txt

### CODE REVIEW

**Comment by dylanmcreynolds** [comment]
I think the gh action failure is because I don't have the ssh key in my forked repo.

File: examples/requirements.txt
Comment (Wiebke) [inline examples/requirements.txt:1]: I additionally needed `ipympl` for `matplotlib` to change its backend to render widgets in Jupyter lab.

File: examples/tiled.ipynb [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    "import matplotlib.animation as animation \n",
+    "from tiled.client import from_uri\n",
+    "from tiled.client.cache import Cache\n",
+    "# from tiled.client import show_logs\n",
+    "import config\n",
+    "\n",
+    "%matplotlib widget\n",
+    "\n",
+    "# displays details of tiled client http traffic\n",
+    "# it's a little wordy but often extremely useful\n",
```
Comment (Wiebke) [inline examples/tiled.ipynb:36]: May be worth to split here to have the import package cell separate from setting up the client.

**Review by Wiebke — APPROVED** [review]
I added commits resolving the previous comments.

**Comment by Wiebke** [comment]
@danton267 @tamidodo Could one of you take a look at the conflicts in requirements.txt? 
Dylan split the file into two such that requirements-dev contains the packages needed only for development. 
I would guess that 
`gunicorn==20.1.0
requests==2.26.0'
would also better fit into requirements-dev.txt as you added these for deployment in [f3989bc](https://github.com/mlexchange/mlex_highres_segmentation/pull/9/commits/f3989bcec9a0b920f3c2e4b8a75bf82a77edb745) and [86d2759](https://github.com/mlexchange/mlex_highres_segmentation/pull/9/commits/86d2759823f021ec9150205ccfe2df982f1a3e30)
Are there any others that are just for development?

**Comment by hannahker** [comment]
@Wiebke both `gunicorn` and `requests` should stay in the `requirements.txt` file. They're required for the deployment on Dash Enterprise and to download sample files from the GDrive, respectively. Looks good to move `black` and `flake8` to `requirements-dev.txt`! 

(We'll definitely remove any dependencies or files necessary for the Dash Enterprise deployment once we have the deployment set up on your infrastructure).

**Pull request merged**